test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -9978,7 +9978,7 @@ public void RestoreNetCore_WithMultipleFrameworksWithPlatformAndAssetTargetFallb
                 var frameworks = new NuGetFramework[]
                 {
                     NuGetFramework.Parse("net5.0-windows10.0.10000.1"),
-                    NuGetFramework.Parse("net5.0-android21")
+                    NuGetFramework.Parse("net5.0-android21.0")
```

Comment:
I thought the . wasn't required for platformversion?

Reply:
It shouldn't be required.  https://github.com/NuGet/Home/issues/9842

Reply:
It's not...it's a limitation of the testing infra I am using...the condition added there is the string passed here, and it needs to match.

Reply:
fixed the infra..should be better now.